modules/fluid_properties/test/tests/brine/tests

```diff
@@ -5,5 +5,19 @@
     csvdiff = 'brine_out.csv'
     allow_test_objects = true
     threading = '!pthreads'
+    requirement = 'BrineFluidProperties shall correctly calculate brine properties'
```

Comment:
"The system shall compute brine properties".  Class name is a product of a design, it is not know when putting requirements together. Correctness is a question of verification and validation - should not be part of a requirement.